config/application.rb

```diff
@@ -17,6 +17,7 @@ class Application < Rails::Application
     config.i18n.fallbacks = true
 
     config.middleware.use "Redirector" unless Rails.env.development?
+    config.middleware.use Rack::Attack
```

Comment:
This looks like state leaking across tests. A good fix is to clear the Rails cache before/after each test. E.g.:

```ruby
setup { Rails.cache.clear }
```

Changing the throttle limits can make the tests pass for now; but it could cause confusing failures as the test suite evolves.